modifyVendor/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="../bootstrap//css//bootstrap.min.css"
+      rel="stylesheet"
+      id="bootstrap-css"
+    />
+    <link href="../styles.css" rel="stylesheet" />
```

Comment:
You can access root directory by doing '/' instead of '../' this is better as it prevents the link from breaking in the future if this module is moved elsewhere